src/components/FormAlertWithSubmitButton.js

```diff
@@ -95,6 +102,19 @@ const FormAlertWithSubmitButton = (props) => {
         );
     }
 
+    if (props.network.isOffline) {
+        return (
+            <View style={[styles.mh5, styles.mb5, styles.flex1, styles.justifyContentEnd, ...props.containerStyles]}>
```

Comment:
thought about refactoring so we didn't have this line twice, but the result was uglier.